internal/config/helpers.go

```diff
@@ -26,6 +26,11 @@ func Environment() string {
 	return "dev"
 }
 
+// EnvironmentPrefix returns the first N characters of Environment.
+func EnvironmentPrefix(n int) string {
+	return Environment()[0:n]
```

Comment:
We should probably test this for the string length. This could panic badly if `len(Environment() < n`).
Can we test this? I am not sure if we mock functions somewhere in the tests, but it should be doable in Go, right?

Reply:
Added test, improved the function.

Reply:
I think a >= will do